src/transfer.ts

```diff
@@ -74,3 +74,53 @@ export async function ensureFunded(funding: EnsureFundedParams, algod: Algodv2,
 
   return undefined
 }
+
+interface TransferAssetParams extends SendTransactionParams {
+  senderAccount: SendTransactionFrom
+  receiverAccount: Account
+  assetId: number
```

Comment:
Worth double checking the algosdk parameter, but I suspect this should be `number | bigint` since asset ID is a 64 bit int (admittedly it will take a long time for it to get big enough to not fit into a `number`

Reply:
I stand corrected, it's just `number`